src/main/play-doc/operation/ClusterBackupRestore.md

```diff
@@ -0,0 +1,68 @@
+# Backup ConductR Cluster
+
+To backup a ConductR cluster use the backup command. The backup command can be used to backup an entire cluster or individual bundles.
+
+## Backing up an entire cluster
+
+The backup includes all the bundles along with their configuration. The backup also includes member and agent configurations.
+
+The following command enables the backup of an entire cluster. The output flag (`-output/-o`) can be used to specify the path to the backup.
+
+```bash
+conduct backup -o backup.zip
+```
+
+The output from the `backup` command can also be redirected to stdout.
+```bash
+conduct backup > backup.zip
+```
+
+## Backing up a bundle
+
+The backup command can also be used to backup a single bundle.
+
+```bash
+conduct backup visualizer > backup.zip
+```
+
+The backup command can take the bundle name or bundle id as input. Both long and short bundle ids are supported.
+If there is ambiguity in bundle name/id, it will result in error similar to other conduct cli commands.
+
+# Restore ConductR Cluster
+
+The restore command can take the output generated by the backup command and restore it to a `ConductR` cluster.
+
+## Restoring from backup
+
+```bash
+conduct restore backup.zip
+```
+
+Once the backup is completed, it can be optionally combined with the restore command.
+
+```bash
+conduct backup | conduct restore -backup -
```

Comment:
Aw, nice! That is what I dreamed of!

How about showing a more interesting use case though and adding a `--ip` to direct the backup to a new cluster? That's what I'd use it for...